Game/model/model.gd

```diff
@@ -1,56 +1,69 @@
-class_name GameModel extends Node
-
+class_name GameModel
+extends Node
 ## This class contains accessors to the major state containers and data for
 ## the complete current state of the game
 
-@onready var player_states: PlayerStates  = %PlayerStates
-@onready var player_spawner := %PlayerSpawner
-@onready var game_state := %GameState
+## When an item quantity is changed, this signal fires
+signal item_count_changed(player_id: int, type: Types.Item, new_count: int)
+## Emitted when ores_layout in PlayerStates is updated.
+signal ores_layout_updated()
+## Emitted when buildings_list in PlayerStates is updated.
+signal buildings_updated()
 
 var world_seed: int
 var player_ids: Array[int]
```

Comment:
ooh. This should not be stored here. Just pull the values off ConnectionSystem instead. If we need to access this often enouigh that we want to stop rebuilding the list over there, just cache it in ConnectionSystem instead of here. Otherwise, you're going to have 2 sources of truth for this value and that's bad.